Tests/HummingbirdFoundationTests/FileMiddlewareTests.swift

```diff
@@ -0,0 +1,52 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2021-2023 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Hummingbird
+@testable import HummingbirdFoundation
+import HummingbirdXCT
+import XCTest
+
+final class FileMiddlewareTests: XCTestCase {
+    func testSearchForIndexHtml() throws {
```

Comment:
Can you move this test to FileTests.swift where all the other FileMiddleware tests are.

Reply:
Wow, I hadn't noticed that. I've made adjustments to align with the existing tests.